js/tween/tweener.babel.js

```diff
@@ -134,11 +135,13 @@ class Tweener {
     @private
   */
   _restorePlayingTweens () {
+    if (this._savedTweens === null) { return; }
```

Comment:
If we have `this._savedTweens = [];` in `_vars`, we don't need to check the presence of the property anymore - it will be always defined. So we need to leave one of them - declaration in the `_vars` or check of presence in `_restorePlayingTweens`, please pick one that works for you but make sure you have covered it with a test. 

Please note that `this._savedTweens === null` equation will cover only `null` case and will not work when `this._savedTweens` is `undefined`, to cover both `null` and `undefined` you want to use `double equal` instead which will allow coercion of types.